src/utils/copyClipboard.js

```diff
@@ -1,7 +1,14 @@
 // Added minor additions
 // Source: https://techoverflow.net/2018/03/30/copying-strings-to-the-clipboard-using-pure-javascript/
+import checkAndroid from './checkAndroid';
+import checkIOS from './checkIOS';
+
+const isMobile = Boolean(checkAndroid() || checkIOS());
```

Comment:
Boolean should not be necessary as checkIOS and checkAndroid both return a boolean already

Reply:
But seeing as it can't hurt it doesn't really matter.

Reply:
No, my bad. Used the checkVersion before this so the Boolean strayed behind. Removing it